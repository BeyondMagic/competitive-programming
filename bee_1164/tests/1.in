4
6
5
28
1
