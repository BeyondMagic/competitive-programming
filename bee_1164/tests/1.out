6 eh perfeito
5 nao eh perfeito
28 eh perfeito
1 nao eh perfeito
